src/test/java/org/humancellatlas/ingest/project/ProjectFilterQueryBuilderTest.java

```diff
@@ -12,7 +12,19 @@
 public class ProjectFilterQueryBuilderTest {
     @Test
     void null_search_type_with_non_null_text() {
-        SearchFilter searchFilter = new SearchFilter("project keyword", null, null, null);
+        SearchFilter searchFilter = new SearchFilter(
+                "project keyword",
+                null,
+                null,
+                null,
+                null,
+                null,
+                null,
+                null,
+                null,
+                null,
+                null
```

Comment:
This looks a bit ugly and you don't really know which null belongs to which parameter. 😒

Reply:
Thanks! Done